shared/test/test_files.rb

```diff
@@ -344,6 +344,38 @@ def test_bogus_metadata
     assert not_found?
   end
 
+  def test_rename_operations
+    filename = @api.randomize_filename('Mixed Case With Space.html')
```

Comment:
The filename choice makes me think it is important and/or part of the test, whereas (I believe) it isn't. Maybe 'Random File.html', which offers the same coverage without the emphasis? Also with `filename2`.